src/components/GuideLayout.tsx

```diff
@@ -0,0 +1,61 @@
+import { FunctionComponent, ReactNode } from 'react'
+
+import { ContentSection } from '@components'
+import { useHubSpot, HubSpotForm } from '@hooks'
+
+interface Form {
+    formId: string
+    onFormSubmitted?: () => void
+}
+
+interface Props {
+    title: string
+    subtitle: string
+    description: ReactNode
+    form: Form
+    formLabel?: string
+    children?: ReactNode
+}
+
+export const GuideLayout: FunctionComponent<Props> = ({ title, subtitle, description, form, formLabel, children }) => {
+    const hubSpotConfig: HubSpotForm = {
+        portalId: '2762526',
+        formId: form.formId,
+        targetId: 'form',
+        formInstanceId: form.formId,
+    }
+    if (form.onFormSubmitted) {
+        hubSpotConfig.onFormSubmitted = form.onFormSubmitted
+    }
+    useHubSpot(hubSpotConfig)
+
+    return (
+        <>
+            <section className="bg-gradient-venus-saturated">
+                <div className="container py-6 d-flex flex-column flex-lg-row justify-content-around align-items-center">
+                    <div className="col-12">
+                        <h1 className="display-2 font-weight-bold mb-4">{title}</h1>
+                        <h4 className="display-3 max-w-750">{subtitle}</h4>
+                    </div>
+                </div>
+            </section>
+
+            <section className="bg-white pt-6 pb-8">
+                <ContentSection className="d-flex flex-column-reverse flex-md-row">
+                    {description}
+
+                    <div className="col-md-6 col-12 pb-md-0 pb-6">
+                        {formLabel && <h2 className="font-weight-bold">{formLabel}</h2>}
```

Comment:
"Download the guide" is included within the form. The repeated title looks odd, so I have made the formLabel optional and removed it in this instance. @bretthayes 
<img width="600" alt="Screen Shot 2022-05-20 at 3 50 41 PM" src="https://user-images.githubusercontent.com/60713139/169608837-a38f2fa7-7887-4b32-978e-eb20b94899de.png">

Reply:
I just went into HubSpot to remove the title in the embedded form so it can stay consistent with the other pages using this layout. I think it'd be great for the title to remain after form submission too. What do you think?

Do you have a HubSpot account? It can be useful to use to update small things like this, CTA verbiage, etc. Greg doesn't want us touching more than that on the forms. `it-tech-ops` can add you to HS and add it to your Okta!

Reply:
Thank you for doing this! I agree that we should go with this pattern. It is added back in here: 0e79ff6

Frankly, I forgot I had access. I checked and do have an account and styling permissions. 

<img width="500" alt="Screen Shot 2022-05-23 at 11 38 35 AM" src="https://user-images.githubusercontent.com/60713139/169867199-c11bd79e-0225-49d3-b6ba-6abbe47bcfb9.png">

Reply:
Solid.update @zlonko @st0nebraker 🎉